src/ValueFormatters/QuantityHtmlFormatter.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace ValueFormatters;
+
+/**
+ * HTML formatter for quantity values.
+ *
+ * @since 0.5.1
+ *
+ * @licence GNU GPL v2+
+ * @author Thiemo Mättig
+ */
+class QuantityHtmlFormatter extends QuantityFormatter {
+
+	/**
+	 * @param string $number
+	 * @param string $unit
+	 *
+	 * @return string HTML
+	 */
+	protected function applyUnit( $number, $unit ) {
+		$html = htmlspecialchars( $number );
```

Comment:
How about requiering the NumberFormatter etc to already return HTML?
And how about having a single method for composing all the bits, where each bit should already be HTML?